Sources/LLBBuildSystemProtocol/DataID.swift

```diff
@@ -6,9 +6,12 @@
 // See http://swift.org/LICENSE.txt for license information
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 
-import llbuild2
 import Foundation
 
+import llbuild2
```

Comment:
should this move to CASProtocol instead? it looks fairly related to the other method you  have

Reply:
Yeah, this whole thing got a little muddled.  I had written it before artifacts but hadn't pushed it.  That'll teach me to leave it unfinished overnight. 😉